Oracle.NoSQL.SDK/src/NoSQLConfig.cs

```diff
@@ -497,6 +497,24 @@ public partial class NoSQLConfig
         /// <seealso cref="SDK.Consistency"/>
         public Consistency Consistency { get; set; } = Consistency.Eventual;
 
+        /// <summary>
```

Comment:
Does the doc tool support something similar to Javadoc \@since to indicate that the API was introduced in a specific release? If so, please add to new methods, etc

Reply:
I couldn't find any xml tag equivalent to @since for C# doc comments.  If you anything, please let me know.